src/app/modules/time-clock/components/entry-fields/entry-fields.component.ts

```diff
@@ -178,4 +200,9 @@ export class EntryFieldsComponent implements OnInit, OnDestroy {
     this.loadActiveEntrySubscription.unsubscribe();
     this.actionSetDateSubscription.unsubscribe();
   }
+
+  isFeatureToggleActivated(): Observable<boolean> {
+    return this.featureManagerService.isToggleEnabledForUser('update-entries')
+      .pipe(map((enabled: boolean) => enabled));
+  }
```

Comment:
Is necessary add a  test when the feature toggle is enable o disable.

you can use this test

```
  const toggleValues = [true, false];
  toggleValues.map((toggleValue) => {
    it(`when FeatureToggle is ${toggleValue} should return ${toggleValue}`, () => {
      spyOn(service, 'isToggleEnabledForUser').and.returnValue(of(toggleValue));

      const isFeatureToggleActivated: Observable<boolean> = component.isFeatureToggleActivated();

      expect(service.isToggleEnabledForUser).toHaveBeenCalled();
      isFeatureToggleActivated.subscribe((value) => expect(value).toEqual(toggleValue));
    });
  });
```

Reply:
Thanks Sandro, I now implement it